.npmrc

```diff
@@ -10,6 +10,7 @@ hoist = false
 public-hoist-pattern[]=*eslint*
 public-hoist-pattern[]=*prettier*
 public-hoist-pattern[]=*stylelint*
+public-hoist-pattern[]=*pdfjs-dist*
```

Comment:
@jonahkagan although the `react-pdf` docs suggest this, I don't think it's necessary, and I think it will require standardizing on a `pdfjs-dist` version through the repo. You can just not hoist and resolve to the root of the package.